QUEST_LV_0100.tsv

```diff
@@ -1,124 +1,124 @@
+QUEST_LV_0100_20150317_000072	$You're good for someone new.{nl}I had a horrible memory there once, so I don't want to go.
+QUEST_LV_0100_20150317_000073	$I'm remembering something...
+QUEST_LV_0100_20150317_000074	$You know that we are dispatched here to defeat Gaigalas right?{nl}Rookie. Can you take this report to Commander Wallis?
+QUEST_LV_0100_20150317_000075	$Commander Wallis
+QUEST_LV_0100_20150317_000076	$Why did you bring this report without Alan coming?{nl}Are you a new soldier?
+QUEST_LV_0100_20150317_000077	$I can return home after we defeat Gaigalas.{nl}Well, I'd like to go home as soon as possible.
+QUEST_LV_0100_20150317_000078	$Wallis is at Bird Footprint Junction.
+QUEST_LV_0100_20150317_000079	$I don't care how good you are. You are new here.{nl}It is an order. Bring some sap from Kepa Raiders.
+QUEST_LV_0100_20150317_000080	$Soldiers these days, they lack guts.{nl}I hope you're a good influence to them.
+QUEST_LV_0100_20150317_000081	$It's clear that we will win this battle.{nl}Why? Because the Owls are helping.
+QUEST_LV_0100_20150317_000082	$The Kepa Raiders are near Ianutis Falls.{nl}To defeat Gaigalas, we will surely need that.
+QUEST_LV_0100_20150317_000083	$Soldier Weisz
+QUEST_LV_0100_20150317_000084	$Hey rookie, are you taking a break?{nl}Get some fat from Duckeys in Lengvas Garden.
+QUEST_LV_0100_20150317_000085	$We will lure Gaigalas using Duckey Fat.{nl}Hey rookie, don't loaf around.
+QUEST_LV_0100_20150317_000086	$Today will be the last day for Gaigalas.{nl}You ready, rookie?
+QUEST_LV_0100_20150317_000087	$Next, we will get leather from Cronewts in Lengvas Garden..{nl}Hurry, soldier.
+QUEST_LV_0100_20150317_000088	$We will win for sure this time.{nl}The Goddesses are on our side.
+QUEST_LV_0100_20150317_000089	$You are pretty good for a rookie.{nl}It's disorganized here at the moment, so you can keep those ingredients for now.{nl}
+QUEST_LV_0100_20150317_000090	$Get to know our soldiers while I make preparations for the battle.{nl}I will call you before we confront Gaigalas.
+QUEST_LV_0100_20150317_000091	$When we burn the Duckey Fat, Gaigalas will appear after smelling the odor.{nl}Lengvas Garden will be a good place to burn it.
+QUEST_LV_0100_20150317_000092	$My heart is pounding.{nl}I feel like my soul is throbbing too.
+QUEST_LV_0100_20150317_000093	$Statue of Old Owl
+QUEST_LV_0100_20150317_000094	$There's no one in this forest. Why are you here?{nl}
+QUEST_LV_0100_20150317_000095	$Gaigalas?{nl}Defeating it without any assistance will be difficult.
+QUEST_LV_0100_20150317_000096	$I have seen many deaths here.{nl}And I don't want to see any more sacrifices.
+QUEST_LV_0100_20150317_000097	$I need Kepa Raider Sap and Cronewt leather.
+QUEST_LV_0100_20150317_000098	$Why do you have that?{nl}Hand it to me and I will make a Fragment of Memory.
+QUEST_LV_0100_20150317_000099	$With this Fragment of Memory I will revive the memories of our sleeping comrades.
```

Comment:
"With this Fragment of Memory I will" -> "With this Fragment of Memory, I will"
Comma.